extensions/wikia/Qualaroo/QualarooHooks.class.php

```diff
@@ -38,10 +38,14 @@ static public function onOasisSkinAssetGroupsBlocking( &$jsAssetGroups ) {
 	}
 
 	static public function onMakeGlobalVariablesScript ( &$vars, $outputPage ) {
-		global $wgNoExternals, $wgQualarooDevUrl, $wgDevelEnvironment, $wgQualarooUrl;
+		global $wgNoExternals, $wgQualarooDevUrl, $wgDevelEnvironment, $wgQualarooUrl, $wgUser, $wgCityId;
 
 		if ( empty( $wgNoExternals ) && $outputPage->getSkin()->skinname == 'oasis' ) {
 			$vars['wgQualarooUrl'] = ($wgDevelEnvironment) ? $wgQualarooDevUrl : $wgQualarooUrl;
+			$vars['isContributor'] = $wgUser->getEditCount() > 0;
+			$vars['isCurrentWikiAdmin'] = in_array( $wgUser->getId(), ( new WikiService() )->getWikiAdminIds() );
```

Comment:
I know getWikiAdminIds() employs cache, though I still recommend to check if this is something we can do on each request